backend/package.json

```diff
@@ -7,7 +7,6 @@
     "build": "tsc --project ./",
     "prestart": "yarn run build",
     "start": "nodemon src/main/App.ts",
-    "test": "jest",
```

Comment:
Wait don't we still need this to run unit tests in BE?

Reply:
See other comment!